src/main/java/no/sikt/nva/pubchannels/channelregistry/model/ChannelRegistryPublisher.java

```diff
@@ -0,0 +1,46 @@
+package no.sikt.nva.pubchannels.channelregistry.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.net.URI;
+import java.util.Optional;
+import no.sikt.nva.pubchannels.Immutable;
+import no.sikt.nva.pubchannels.channelregistry.mapper.ScientificValueMapper;
+import no.sikt.nva.pubchannels.handler.ScientificValue;
+import no.sikt.nva.pubchannels.handler.ThirdPartyPublisher;
+
+@JsonSerialize
+public record ChannelRegistryPublisher(@JsonProperty(IDENTIFIER_FIELD) String identifier,
+                                       @JsonProperty(LEVEL_FIELD) ChannelRegistryLevel channelRegistryLevel,
+                                       @JsonProperty(ISBN_PREFIX_FIELD) String isbnPrefix,
+                                       @JsonProperty(NAME_FIELD) String name,
+                                       @JsonProperty(HOMEPAGE_FIELD) URI homepage,
+                                       @JsonProperty(DISCONTINUED) String discontinued)
+    implements Immutable, ThirdPartyPublisher {
+
+    private static final String IDENTIFIER_FIELD = "pid";
+    private static final String NAME_FIELD = "name";
+    private static final String ISBN_PREFIX_FIELD = "isbnprefix";
+    private static final String LEVEL_FIELD = "levelElementDto";
+    private static final String HOMEPAGE_FIELD = "kurl";
+    public static final String DISCONTINUED = "ceased";
+
+    @Override
+    public String getYear() {
```

Comment:
Do you need @JsonIgnore on java records getters to prevent them from getting serialised?